components/hint-message/hint-message.jsx

```diff
@@ -24,7 +24,10 @@ class HintMessage extends React.Component {
 HintMessage.propTypes = {
   header: PropTypes.string.isRequired,
   iconComponent: PropTypes.element.isRequired,
-  linkComponent: PropTypes.element.isRequired
+  linkComponent: PropTypes.oneOfType([
+    PropTypes.bool,
```

Comment:
the HintMessage code already fully expected this to be possible (see [here](https://github.com/mozilla/network-pulse/blob/edb1b5170c5bb0a994cbec8a2ee4be67000db230/components/hint-message/hint-message.jsx#L7)) but for some reason the proptypes weren't set up to allow for that situation.